io.github.PopSearch.json

```diff
@@ -0,0 +1,155 @@
+{
+    "app-id": "io.github.PopSearch",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "47",
+    "sdk": "org.gnome.Sdk",
+    "command": "/app/bin/search_bar.sh",
+    "finish-args": [
+        "--share=network",
+        "--socket=wayland",
+        "--socket=fallback-x11",
+        "--share=ipc",
+        "--filesystem=home"
+    ],
+    "modules": [
+        {
+            "name": "PopSearch",
+            "buildsystem": "simple",
+            "build-commands": [
+                "install -Dm755 src/search-bar.py /app/bin/search-bar.py",
+                "install -Dm755 src/search_bar.sh /app/bin/search_bar.sh",
+                "install -Dm644 src/data/PopSearch.desktop /app/share/applications/io.github.PopSearch.desktop",
+                "install -Dm644 src/data/io.github.PopSearch.png /app/share/icons/hicolor/256x256/apps/io.github.PopSearch.png"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/rajnisht7/PopSearch-Release/archive/refs/tags/v1.0.2.tar.gz",
+                    "sha256": "a371d81cfb2770e4bcf5c5e2c8bcfcffcd26e93e7e82b2bebeb05c9c5e36702c",
+                    "x-checker-data": {
+                        "type": "github",
+                        "user": "rajnisht7",
+                        "repo": "PopSearch-Release",
+                        "tag-pattern": ".*"
+                    }
+                }
+            ]
+        },
+        {
+    "name": "python3-modules",
```

Comment:
the indentation is off, making it hard. Also if you use the flatpak-builder-tools this is usually a valid case of "including" a separate manifest was it is auto generated (instead of inlining).